tests/target/issue-5193/match_block_trailing_comma_true/trailing_comma_never.rs

```diff
@@ -0,0 +1,16 @@
+// rustfmt-max_width: 90
+// rustfmt-match_block_trailing_comma: true
+// rustfmt-trailing_comma: Never
+
+fn main() {
+    match FooBar::Foo {
+        FooBar::Foo => unreachable!(),
+        FooBar::Bar => {
+            println!("match_block_trailing_comma: true");
+            println!("trailing_comma: Never");
+        },
+        FooBar::Baz => {
+            println!("Lorem ipsum dolor sit amet, consectetuer adipiscing elit.")
+        }
```

Comment:
I find it a little odd that `trailing_comma=Never` takes precedence over `match_block_trailing_comma=true`, for the last match arm but that's what I needed to do in order to prevent the idempotency issue since that's how `arm_comma` is implemented.

https://github.com/rust-lang/rustfmt/blob/a67d909627b7ec93e315bfe1c66895904055c75c/src/matches.rs#L146-L160